lib/Word_Lib/Boolean_Inequalities.thy

```diff
@@ -129,12 +129,10 @@ lemma
   "x XOR y \<le> NOT x OR NOT y" for x :: "'a::len word"
   by (rule word_bool_le_funs)
 
-declare word_bool_le_funs[intro!]
-
```

Comment:
Very minor, but if the intention is to not have these rules available to the simplifier then you might as well remove this `declare` from the previous commit.

Reply:
Also done.